appcell-resources/js/common.js

```diff
@@ -398,6 +425,12 @@ cm.createSideMenu = function() {
 //        $(".slide-menu").removeClass('slide-on');
         cm.toggleSlide();
     });
+    $("#b-setlng-ok").on('click', function() {
+        $("#selectLng option:selected").each(function(index, option) {
+            i18next.changeLanguage($(option).val())
+            window.location.reload();
```

Comment:
Some of the contents are not in data-i18n attribute of the HTML tags. Therefore, I must call window.reload. I will try to fix it later.